staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -1007,11 +1007,11 @@ func (r *Request) transformResponse(resp *http.Response, req *http.Request) Resu
 func truncateBody(body string) string {
 	max := 0
 	switch {
-	case bool(klog.V(10)):
+	case bool(klog.V(5)):
 		return body
-	case bool(klog.V(9)):
+	case bool(klog.V(4)):
 		max = 10240
-	case bool(klog.V(8)):
+	case bool(klog.V(3)):
 		max = 1024
 	}
```

Comment:
/hold

on this change... trace-level request logging is intentionally at much higher levels. mixing request-level trace logs with logic-level trace logs ("did x", "did y", etc) would make the logic-level trace logs almost impossible to consume